src/components/GeneralModal/styles.js

```diff
@@ -26,7 +26,6 @@ const useStyles = makeStyles(() => ({
       fontFamily: 'Poppins',
       color: 'gray',
       fontSize: '16px',
-      textTransform: 'capitalize',
```

Comment:
this property could be better with value textTransform: toLowerCase

Reply:
Done